bitnami/solr/README.md

```diff
@@ -0,0 +1,347 @@
+| `exporter.image.tag`                            | Solr exporter image tag                                                                                     | `{TAG_NAME}`                                                                    |
+| `exporter.image.pullPolicy`                     | Image pull policy                                                                                           | `IfNotPresent`                                                                  |
+| `exporter.image.pullSecrets`                    | Specify docker-registry secret names as an array                                                            | `nil`                                                                           |
+| `exporter.configFile`                           | Config file for the Solr prometheus exporter                                                                | `/opt/bitnami/solr/contrib/prometheus-exporter/conf/solr-exporter-config.xml`   |
+| `port`                                          | Solr exporter port                                                                                          | `9983`                                                                          |
+| `threads`                                       | Number of Solr exporter Threads                                                                             | `7`                                                                             |
+| `exporter.podLabels`                            | Additional labels for Metrics exporter pod                                                                  | {}                                                                              |
+| `exporter.podAnnotations`                       | Additional annotations for Metrics exporter pod                                                             | {}                                                                              |
+| `exporter.resources`                            | Exporter resource requests/limit                                                                            | Memory: `256Mi`, CPU: `100m`                                                    |
+| `exporter.service.type`                         | Kubernetes Service type (solr metrics)                                                                     | `ClusterIP`                                                                     |
+| `exporter.service.annotations`                  | Annotations for the services to monitor.                                                                    | {}                                                                              |
+| `exporter.service.labels`                       | Additional labels for the metrics service                                                                   | {}                                                                              |
```

Comment:
No reference to the tls section here. Ran the script (ignore the false positives):

```
ERROR: Value service.nodePorts.http is not in README.md
ERROR: Value service.nodePorts.https is not in README.md
ERROR: Value cloudBootstrap is not in README.md
ERROR: Value podLabels is not in README.md
ERROR: Value podAnnotations is not in README.md
ERROR: Value lifecycleHooks is not in README.md
ERROR: Value volumePermissions.containerSecurityContext is not in README.md
ERROR: Value tls.enabled is not in README.md
ERROR: Value tls.wantClientAuth is not in README.md
ERROR: Value tls.needClientAuth is not in README.md
ERROR: Value tls.keystorePassword is not in README.md
ERROR: Value tls.importKubernetesCA is not in README.md
ERROR: Value tls.checkPeerName is not in README.md
ERROR: Value tls.caSecret.name is not in README.md
ERROR: Value tls.caSecret.bundlePath is not in README.md
ERROR: Value tls.certSecret.name is not in README.md
ERROR: Value tls.certSecret.keyPath is not in README.md
ERROR: Value tls.certSecret.certPath is not in README.md
ERROR: Value persistence.annotations is not in README.md
ERROR: Value persistence.mountPath is not in README.md
ERROR: Value zookeeper.service.publishNotReadyAddresses is not in README.md
ERROR: Value exporter.port is not in README.md
ERROR: Value exporter.threads is not in README.md
ERROR: Value exporter.containerSecurityContext is not in README.md
ERROR: Value exporter.podSecurityContext is not in README.md
ERROR: Value exporter.updateStrategy is not in README.md
ERROR: Value exporter.service.port is not in README.md
ERROR: Value exporter.service.nodePorts.http is not in README.md
ERROR: Value exporter.service.nodePorts.https is not in README.md
ERROR: Value exporter.service.loadBalancerSourceRanges is not in README.md
ERROR: Value exporter.hostAliases is not in README.md
ERROR: Value ingress.pathType is not in README.md
ERROR: Value ingress.apiVersion is not in README.md
```